<?xml version="1.0"?>
<launch>
    <include file="$(find img_input)/launch/img_input.launch" />

    <node pkg="v10_pillar_detector" type="v10_pillar_detector_node" name="v10_pillar_detector_node" output="screen" >
<!--             <param name="variance" type="double" value="2.0" />    -->
        <remap from="/v10_pillar_detector_node/image_in" to="/img_input_node/image_out" />
        <!-- <remap from="/v10_pillar_detector_node/camera_info_in" to="/usb_cam/camera_info" /> -->
        <remap from="/v10_pillar_detector_node/frame_mode" to="/pillar_monitor_node/frame_mode" />
        <remap from="/v10_pillar_detector_node/LUT_data" to="/pillar_monitor_node/LUT_data" />
        <remap from="/v10_pillar_detector_node/save_param" to="/pillar_monitor_node/save_param" />
        <param name="sign_config_path" type="string" value="$(find v10_pillar_detector)/config/saved_config.yaml" />
    </node>
</launch>
